labs/lab08-airgap-lite/README.md

```diff
@@ -1,111 +1,106 @@
-Lab 1.8: Airgap
+Lab 1.8: Air gap
 =========================================
 
-In this lab, we'll review how to perform installations in Air Gap environments, and
-how to collect support bundles in Air Gap environments.
+In this lab, we'll review how to perform installations and collect support bundles in Air Gap environments.
 
 * **What you will do**:
     * Access and verify a single-node Air Gap setup via a bastion server
-    * Learn to use KOTS to install in an Air Gap environment
+    * Learn how to install in an Air Gap environment using app manager
     * Create an SSH tunnel to configure an Air Gap instance
     * Perform an upgrade of an application in an Air Gap environment
     * Use the `kubectl support-bundle` CLI in an Air Gap environment
-* **Who this is for**: This lab is for anyone who builds/maintains KOTS applications (see note below)
-    * Full Stack / DevOps / Product Engineers
+
```

Comment:
Is there a reason to add a newline? Markdown rendering isn't going to add a newline.

Reply:
I was doing it to stay consistent with the other items in the list. But it's fine if we want to keep it as is